transaction/core/src/blockchain/block.rs

```diff
@@ -344,17 +344,17 @@ mod block_tests {
         assert_eq!(
             block.id.as_ref(),
             &[
-                54, 149, 234, 64, 10, 166, 9, 176, 203, 154, 166, 123, 16, 98, 112, 231, 21, 243,
-                198, 135, 186, 238, 195, 29, 166, 59, 85, 50, 215, 137, 171, 104
+                118, 205, 187, 34, 207, 104, 52, 137, 97, 124, 79, 205, 112, 204, 146, 217, 128,
+                178, 169, 214, 231, 120, 46, 237, 17, 93, 59, 136, 101, 131, 197, 217
```

Comment:
I wonder if we should change the way this test is working, and make the block contents be "old" block contents so that we don't change the hashes here.

We could separately test block hashes that do have memos